chromium_src/third_party/abseil-cpp/absl/random/random.h

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_ABSEIL_CPP_ABSL_RANDOM_RANDOM_H_
+#define BRAVE_CHROMIUM_SRC_THIRD_PARTY_ABSEIL_CPP_ABSL_RANDOM_RANDOM_H_
+
+#include "src/third_party/abseil-cpp/absl/random/random.h"
+
+namespace brave {
```

Comment:
Since this is being inserted into absl I think we should keep it in the absl namespace and give it a generic name like RandenEngine, but feel free to typedef it in brave code with the name you're using everywhere else

Reply:
I agree about `absl` namespace, don't need to introduce `brave` here. how about this?
```
namespace absl {

// Make randen_engine available for direct usage, because
// absl::BitGen/InsecureBitGen uses always-on process-bound salt for seeded
// generators.
template <typename T>
using randen_engine = random_internal::randen_engine<T>;

}  // namespace absl
```

Reply:
another option might be to add non-salted seed sequence support, but I'm not sure how properly do this. maybe something like that:
```
// need to come up with BraveUnsaltedSeedSeq
// and then

template <typename T>
struct is_salted_seed_seq<
    T, typename std::enable_if<std::is_same<T, BraveUnsaltedSeedSeq>::type>
    : public std::true_type {};
```

Reply:
Implemented the `using randen_engine` solution in https://github.com/brave/brave-core/pull/13118/commits/fd01f99afd2efa12c1e9bf20c440176160433b94